doc/extractors/CNFGateFeatures.md

```diff
@@ -1,5 +1,5 @@
 # Gate Features for CNF Instances
 
-- Context: cnf
+**Valid contexts**: `cnf`
 
-Runs the gate feature extraction algorithm described in~\cite{} and extracts features denoting the numbers of gates of specific types and their distribution of the levels of the detected hierarchical gate structure.
\ No newline at end of file
+Runs the gate feature extraction algorithm described [here](!!Missing!!) and extracts features denoting the numbers of gates of specific types and their distribution of the levels of the detected hierarchical gate structure.
```

Comment:
Note the missing link here that should still be fixed.

Reply:
will do